### PR TITLE
tests, utils: Extract GetVcpuMask func as a private func

### PR DESCRIPTION
**Pull request by Barakmor1**
Move GetVcpuMask func as a private func to vmi_configuration_test.go. 
Since it is the only place it is being called.

### What this PR does
Before this PR:

After this PR:

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/vmi_configuration_test.go [hunk 1]
```diff
@@ -3391,3 +3391,22 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
 
 	return virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dataVolume, metav1.CreateOptions{})
 }
+
+func getVcpuMask(pod *k8sv1.Pod, emulator, cpu string) (output string, err error) {
+	BinBash := "/bin/bash"
```
Comment (orelmisan) [inline tests/vmi_configuration_test.go:3396]: nit: This could be a const and start with a lowercase letter.

**Review by nunnatsa — APPROVED** [review]

**Review by orelmisan — COMMENTED** [review]
Thank you for the cleanup @Barakmor1
Just one nit.

File: tests/vmi_configuration_test.go (hunk 1 above)
Comment (Barakmor1) [reply tests/vmi_configuration_test.go]: Done

**Review by dankenigsberg — APPROVED** [review]

**Review by EdDev — COMMENTED** [review]
> move GetVcpuMask func
> 
> as a private func to
> vmi_configuration_test.go.
> Since it is the only place
> it is being called.

Please format the commit message a bit.
The recommendation is something like this:

```
<topic>: <subject>

<body>
```

I am not sure why the lines are so short, but you can wrap text at 72 chars [1] or something close.

I usually use the `topic` as the component or location of the change. In this case, it could be `tests, utils:`.
[1] https://cbea.ms/git-commit/#wrap-72

**Review by EdDev — CHANGES_REQUESTED** [review]
Please fix the compilation error and adjust the commit format.

**Comment by Barakmor1** [comment]
Thanks, i updated the commit massage.

File: tests/vmi_configuration_test.go [hunk 2]
```diff
@@ -3391,3 +3391,21 @@ func createBlockDataVolume(virtClient kubecli.KubevirtClient) (*cdiv1.DataVolume
...
+	args := []string{"/bin/bash", "-c", pscmd}
+	Eventually(func() error {
+		output, err = exec.ExecuteCommandOnPod(virtClient, pod, "compute", args)
+		return err
+	}).Should(Succeed())
+	vcpupid := strings.TrimSpace(strings.Trim(output, "\n"))
+	tasksetcmd := "taskset -c -p " + vcpupid + " | cut -f2 -d:"
+	args = []string{"/bin/bash", "-c", tasksetcmd}
+	output, err = exec.ExecuteCommandOnPod(virtClient, pod, "compute", args)
+	Expect(err).ToNot(HaveOccurred())
```
Comment (EdDev) [inline tests/vmi_configuration_test.go:3408]: The move itself if fine, but there seems to be other things that merits cleaning in the function. Please consider fixing those in another commit or PR as well.

**Review by EdDev — APPROVED** [review]

**Review by ShellyKa13 — COMMENTED** [review]
/lgtm

**Comment by Barakmor1** [comment]
@ShellyKa13 Hey 
I had to rebase could you please have another look :)?

**Review by dankenigsberg — APPROVED** [review]

**Pull request merged**